src/Core/Prefab/GlobeView.js

```diff
@@ -427,13 +428,25 @@ GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFro
     direction.applyMatrix4(matrix);
     direction.sub(ray.origin);
 
-    var angle = direction.angleTo(ray.direction);
+    const angle = direction.angleTo(ray.direction);
 
     depthRGBA.fromArray(buffer).divideScalar(255.0);
 
-    var depth = unpack1K(depthRGBA, 100000000.0) / Math.cos(angle);
+    let orthoZ = 0;
```

Comment:
Maybe you could move this function to `c3DEngine`, something like: `c3DEngine.depthBufferValueToZ(depthRGBA)`?

Reply:
done